app/src/main/res/layout/activity_qrexchange.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <android.support.design.widget.AppBarLayout
+        android:id="@+id/app_bar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fitsSystemWindows="true"
+        android:theme="@style/AppTheme.AppBarOverlay">
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/toolbar_title"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:textAlignment="center"
+                android:gravity="center"
+                android:layout_weight="1"
+                android:text="@string/app_name"
+                android:textSize="@dimen/toolbar_title_size"
+                android:textStyle="bold" />
+            <ImageButton
+                android:id="@+id/backButtonQRExchange"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:backgroundTint="@color/colorPrimary"
+                android:src="?attr/actionModeCloseDrawable"/>
+
+
+
+        </FrameLayout>
+
+    </android.support.design.widget.AppBarLayout>
+
+
+    <Button
+        android:id="@+id/scanbutton"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Scan QR code" />
```

Comment:
Can you put this in strings.xml? Otherwise the Dutch translation isn't reflected on this page